tests/sqllogictests/suites/base/05_ddl/05_0036_sequence.test

```diff
@@ -4,5 +4,69 @@ DROP SEQUENCE IF EXISTS seq
 statement ok
 CREATE SEQUENCE seq
 
+statement ok
+DROP TABLE IF EXISTS tmp;
+
+statement ok
+DROP TABLE IF EXISTS tmp1;
+
+statement ok
+DROP TABLE IF EXISTS tmp2;
+
+statement ok
+DROP TABLE IF EXISTS tmp3;
+
+statement ok
+CREATE TABLE tmp1(a string);
+
+statement ok
+CREATE TABLE tmp2(a uint64);
+
+statement ok
+CREATE TABLE tmp3(a int, b uint64, c int);
+
+statement ok
+INSERT INTO tmp1 values(nextval(seq));
+
+statement ok
+INSERT INTO tmp2 values(nextval(seq));
+
+statement ok
+INSERT INTO tmp2 values(nextval(seq));
+
+statement ok
+INSERT INTO tmp2 values(nextval(seq)), (1024);
+
+query I
+select * from tmp2 order by a;
+----
+2
+3
+4
+1024
+
+statement ok
+INSERT INTO tmp3 values(1,nextval(seq),3);
+
+query III
+select * from tmp3;
+----
+1 5 3
+
 statement ok
 DROP SEQUENCE seq
+
+statement error 1126
+DROP SEQUENCE seq
+
+statement error 3101
+INSERT INTO tmp3 values(1,nextval(seq),3);
```

Comment:
how to optimize query:

```
INSERT INTO tmp3 select 1,nextval(seq) from numbers(10000)
```

Reply:
Another question (not mentioned in the PR Summary): How does `nextval` function work in a distributed clsuter?

Reply:
add new `RelOperator::AsyncFunction`, it distrubute mode is `serial`.

add a transformer `transform_sequence_nextval.rs`, it will generate sequence base on number rows of input block, hence not query meta for many time:
```
        let count = data_block.num_rows() as u64;
        let req = GetSequenceNextValueReq {
            ident: SequenceIdent::new(&tenant, &self.sequence),
            count,
        };
```